riberry/i2c_base.py

```diff
@@ -72,7 +75,12 @@ def i2c_write(self, packet):
             print(e)
             return
         try:
-            self.i2c.writeto(self.i2c_addr, packet)
+            if isinstance(self.i2c, i2c):
+                self.i2c.write(packet)
+            elif isinstance(self.i2c, busio.I2C):  # NOQA
```

Comment:
An error was occurring here without NOQA, and removing NOQA eliminates the error, but it also causes the program to fail to run correctly. It’s important to note that NOQA should not be added indiscriminately.